src/alembic/versions/e74292feb010_add_profile_picture_url_to_site_user.py

```diff
@@ -0,0 +1,34 @@
+"""add profile_picture_url to site_user
+
+Revision ID: e74292feb010
+Revises: 2a6ea95342dc
+Create Date: 2024-10-04 17:23:06.869439
+
+"""
+from collections.abc import Sequence
+from typing import Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "e74292feb010"
+down_revision: str | None = "2a6ea95342dc"
+branch_labels: str | Sequence[str] | None = None
+depends_on: str | Sequence[str] | None = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("fk_officer_term_computing_id_site_user", "officer_term", type_="foreignkey")
+    op.create_foreign_key(op.f("fk_officer_term_computing_id_user_session"), "officer_term", "user_session", ["computing_id"], ["computing_id"])
```

Comment:
This foreign key constraint update doesn't look right, since you haven't modified any foreign keys. I'm not sure why it autogenerated...

ah! I see the bug is an inconsistency between the alembic migration & talbes.py for officers; I'll fix it now

Reply:
done https://github.com/CSSS/csss-site-backend/pull/89/commits/652ed1c768adb3e664449eea59f535acdaf81326

The issue was that I guess I wasn't very familiar with foreign keys a few months ago and got confused.

Reply:
ooooooo ok ty for fixing!